<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>785</width>
    <height>607</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>HJ Editor</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="CodeEditor" name="editor">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>761</width>
      <height>431</height>
     </rect>
    </property>
   </widget>
   <widget class="Console" name="outputText">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>440</y>
      <width>761</width>
      <height>91</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>785</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="actionNewFile"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave_File"/>
    <addaction name="action_Unicode"/>
    <addaction name="action_ANSI"/>
   </widget>
   <widget class="QMenu" name="menuEdit_O">
    <property name="title">
     <string>编辑</string>
    </property>
    <addaction name="actionSelectAll"/>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
    <addaction name="actionFind"/>
   </widget>
   <widget class="QMenu" name="menuHelp_H">
    <property name="title">
     <string>帮助</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionDocument"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menuEdit_O"/>
   <addaction name="menuHelp_H"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="iconSize">
    <size>
     <width>24</width>
     <height>24</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonIconOnly</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNewFile"/>
   <addaction name="actionOpen"/>
   <addaction name="actionSave_File"/>
   <addaction name="actionUndo"/>
   <addaction name="actionRedo"/>
   <addaction name="actionRun"/>
   <addaction name="actionSetting"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionNewFile">
   <property name="icon">
    <iconset resource="image.qrc">
     <normaloff>:/image/New.png</normaloff>:/image/New.png</iconset>
   </property>
   <property name="text">
    <string>新建</string>
   </property>
   <property name="toolTip">
    <string>新建一个C++代码文件</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="image.qrc">
     <normaloff>:/image/Open.png</normaloff>:/image/Open.png</iconset>
   </property>
   <property name="text">
    <string>打开...</string>
   </property>
   <property name="toolTip">
    <string>打开文件...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSelectAll">
   <property name="text">
    <string>全选</string>
   </property>
   <property name="toolTip">
    <string>选择全部文本</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="icon">
    <iconset resource="image.qrc">
     <normaloff>:/image/Undo.png</normaloff>:/image/Undo.png</iconset>
   </property>
   <property name="text">
    <string>撤销</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="icon">
    <iconset resource="image.qrc">
     <normaloff>:/image/Redo.png</normaloff>:/image/Redo.png</iconset>
   </property>
   <property name="text">
    <string>重做</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+Z</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>关于 Fresh-Editor</string>
   </property>
   <property name="toolTip">
    <string>关于 Fresh-Editor</string>
   </property>
  </action>
  <action name="actionDocument">
   <property name="text">
    <string>说明文档</string>
   </property>
   <property name="toolTip">
    <string>查看说明文档</string>
   </property>
  </action>
  <action name="actionSave_File">
   <property name="icon">
    <iconset resource="image.qrc">
     <normaloff>:/image/Save.png</normaloff>:/image/Save.png</iconset>
   </property>
   <property name="text">
    <string>保存文件</string>
   </property>
   <property name="toolTip">
    <string>保存文件</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionRun">
   <property name="icon">
    <iconset resource="image.qrc">
     <normaloff>:/image/Run.png</normaloff>:/image/Run.png</iconset>
   </property>
   <property name="text">
    <string>运行</string>
   </property>
   <property name="toolTip">
    <string>编译并运行代码</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionSetting">
   <property name="icon">
    <iconset resource="image.qrc">
     <normaloff>:/image/Setting.png</normaloff>:/image/Setting.png</iconset>
   </property>
   <property name="text">
    <string>设置</string>
   </property>
   <property name="toolTip">
    <string>更改编辑器的设置</string>
   </property>
   <property name="visible">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionFind">
   <property name="icon">
    <iconset resource="image.qrc">
     <normaloff>:/image/Find.png</normaloff>:/image/Find.png</iconset>
   </property>
   <property name="text">
    <string>查找</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="action_Unicode">
   <property name="text">
    <string>保存为Unicode</string>
   </property>
  </action>
  <action name="action_ANSI">
   <property name="text">
    <string>保存为ANSI</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>CodeEditor</class>
   <extends>QPlainTextEdit</extends>
   <header location="global">codeeditor.h</header>
  </customwidget>
  <customwidget>
   <class>Console</class>
   <extends>QPlainTextEdit</extends>
   <header location="global">console.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="image.qrc"/>
 </resources>
 <connections/>
</ui>
